webapp/src/Controller/Jury/ContestController.php

```diff
@@ -855,4 +855,65 @@
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/{contestId<\d+>}/request-remaining", name="jury_contest_request_remaining")
+     */
+    public function requestRemainingRunsWholeContest(int $contestId): RedirectResponse
+    {
+        /** @var Contest $contest */
+        $contest = $this->em->getRepository(Contest::class)->find($contestId);
+        if (!$contest) {
+            throw new NotFoundHttpException(sprintf('Contest with ID %s not found', $contestId));
+        }
+        $judgings = [];
+        foreach ($contest->getProblems() as $problem) { // This should get the ContestProblem
+            foreach ($problem->getSubmissions() as $submission) { // So this has only submissions in this contest
```

Comment:
These loops will result in 1 (contest) + N (problems) + M (submissions) + T (teams) + J (judgings) queries. Maybe we should, at line 865, use a query builder to fetch these rows in one query?